bolt-design-system/bolt
# Minor Stability Updates

## Jira
N/A

## Summary
Updates the Lerna config to not ignore the SSR server folder (nohoist) + adds a new "warmup" function to the image task as a possible workaround to the sharp SVG-related build errors that are still periodically occurring.

## How to test
Confirm build passes + evaluate consistency of yarn installs / overall builds not randomly failing when running the image build task.
